examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,218 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Api.Gax;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    using Lib;
+    using Google.Ads.GoogleAds.V3.Errors;
+    using Google.Ads.GoogleAds.V3.Services;
+    using System;
+
+    /// <summary>
+    /// This example gets the account hierarchy of the specified manager account. If you don't specify
+    /// manager customer ID, the example will instead print the hierarchies of all accessible customer
+    /// accounts for your authenticated Google account.
+    /// Note that if the list of accessible customers for your authenticated Google account includes
+    /// accounts within the same hierarchy, this example will retrieve and print the overlapping
+    /// portions of the hierarchy for each accessible customer.
+    /// </summary>
+    public class GetAccountHierarchy : ExampleBase
+    {
+        // private const string MANAGER_CUSTOMER_ID = "3955750366";//"INSERT_MANAGER_CUSTOMER_ID_HERE";
```

Comment:
This commented out line looks like it needs some changes. Perhaps it was left in from testing? I think this should also be in the Main if you want people to use it.

Reply:
Correct, fixed.